distribution/docs/src/main/jdocs/content/_configuring/configuring-managed-service-factory-bundles-contents.adoc

```diff
@@ -32,58 +31,87 @@ The following table shows the format to use for each property type supported.
+|name=L"value"
+|`diameter=L"10000"`
 
 |Floats
-|name = F"value"
-|cost = F"10.50"
+|name=F"value"
+|`cost=F"0x41280000"`
 
 |Doubles
-|name = D"value"
-|latitude = D"45.0234"
+|name=D"value"
+|`latitude=D"0x423417f6"`
 
 |Lists of Strings
-|name = [ "value1", "value2", …​ ]
-|`complexStringArray = [
-    "{\"url\"\ \"http://test.sample.com\"\ \"layers\"\ [\"0\"]\
-    \"VERSION\"\ \"1.1\|1.2\"\ \"image/png\"}\ \"beta\"\ 1}",	"{\"url\"\ \"http://test.sample.com"\ 0.5}",
-	"/solr\=SAML\|PKI\|basic", "/security-config\=SAML\|basic"
-]`
+|name=["value1","value2",…​]
+|`complexStringArray=["{\"url\"\ \"http://test.sample.com\"\ \"layers\"\ [\"0\"]\ \"VERSION\"\ \"1.1\|1.2\"\ \"image/png\"}\ \"beta\"\ 1}","{\"url\"\ \"http://test.sample.com"\ 0.5}","/solr\=SAML\|PKI\|basic","/security-config\=SAML\|basic"]`
+
+|Lists of Booleans
+|name=B["true\|false","true\|false",…​]
+|`authorizedList=B["true","false"]`
 
 |Lists of Integers
-|name = I[ "value1", "value1", …​ ]
-|sizes = I[ "10", "20", "30" ]
+|name=I["value1","value2",…​]
+|`sizes=I["10","20","30"]`
+
+|Lists of Longs
+|name=L["value1","value2",…​]
+|`sizes=L["100","200","300"]`
+
+|Lists of Floats
+|name=F["value1","value2",…​]
+|`sizes=F["0x3f8ccccd","0x400ccccd","0x40533333"]`
+
+|Lists of Doubles
+|name=D["value1","value2",…​]
+|`sizes=D["0x3f8fbe77","0x400ef9db","0x4056147b"]`
+
+|Vectors of Strings
+|name=("value1","value2",…​)
+|`names=("A","B","C")`
+
+|Vectors of Booleans
+|name=B("true\|false","true\|false",…​)
+|`authorizedVector=B("true","false")`
+
+|Vectors of Integers
+|name=I("value1","value2",…​)
+|`sizes=I("10","20","30")`
+
+|Vectors of Longs
+|name=L("value1","value2",…​)
+|`sizes=L("100","200","300")`
+
+|Vectors of Floats
+|name=F("value1","value2",…​)
+|`sizes=F("0x3f8ccccd","0x400ccccd","0x40533333")`
+
+|Vectors of Doubles
+|name=D("value1","value2",…​)
+|`sizes=D("0x3f8fbe77","0x400ef9db","0x4056147b")`
 
 |===
 
 [NOTE]
 ====
-* Lists of values can be prefixed with any of the supported types (B, I, L, F or D).
-* To prevent any configuration issues, the = signs used in values should be escaped using a backslash (\).
-* Boolean values will default to false if any value other than true is provided.
-* Escape character in values must be used for double quotes (") and spaces, but cannot be used with { } or [ ] pairings.
+* Lists and Vectors of values can be prefixed with any of the supported types (`B`, `I`, `L`, `F` or `D`).
+* To prevent any configuration issues, the `=` signs used in values should be escaped using a backslash (`\`).
+* Boolean values will default to `false` if any value other than `true` is provided.
+* Escape character in values must be used for double quotes (`"`) and spaces, but cannot be used with `{` `}` or `[` `]` pairings.
+* Floats and Longs must be written in the IEEE 754 format. For example, the example for the Floats and Doubles in the table above correspond to 10.50 and 45.0234, respectively.
 ====
 
 .Sample configuration file
```

Comment:
✏️  Since we say that configuration files shouldn't contain `service.pid`, we should update all the configuration files in the documentation, including this sample one and the one below.